templates/lookup/ifupdown__combined_interfaces.j2

```diff
@@ -67,7 +67,11 @@
 {%       endif %}
 {%     endif %}
 {%     if params.allow is undefined %}
-{%       set _ = params.update({"allow":["hotplug"]}) %}
+{%       if (ansible_virtualization_role == 'guest' and ansible_virtualization_type in [ 'lxc', 'openvz' ]) %}
+{%         set _ = params.update({"allow":["boot"]}) %}
```

Comment:
Ehh, I've written a long rant about this change backported to Debian Stable being a mistake, but oh well. Live and let live, I suppose that it was a bug that prevented the boot of a system and that's much more severe than issues with setting up virtual network interfaces - these could be dealt with by the `networking.service` unit. I talked a bit about this [on the Debian systemd maintainers mailing list](https://lists.alioth.debian.org/pipermail/pkg-systemd-maintainers/2017-February/014355.html), I suspect that if the Debian Developers go ahead with the idea of modifying the `networking.service` to use separate `systemd` cgroups this issue will show up again. I wonder what solution they will come up with.